server/models/ipfspins.ts

```diff
@@ -0,0 +1,40 @@
+import * as Sequelize from 'sequelize';
```

Comment:
Can we rename this file `ipfs_pins.ts` to stick with the standard style?

Reply:
Done